.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,4 @@
 **Before merging:**
 
 - [ ] Copy the latest benchmark results into the `README.md` and update this PR
+- [ ] Add this change to `CHANGELOG.md` following Keep a Changelog [guidelines](https://keepachangelog.com/en/1.0.0/) with [semantic versioning](https://semver.org/)
```

Comment:
Since many PRs won't really merit a CHANGELOG entry, could this be "Consider whether this change merits an update to `CHANGELOG.md`...",  or something like that?